js/base/Exchange.js

```diff
@@ -2361,36 +2361,33 @@ module.exports = class Exchange {
         return this.filterBySymbolSinceLimit (sorted, symbol, since, limit);
     }
 
-    isPostOnly (type, timeInForce = undefined, exchangeSpecificOption = undefined, params = {}) {
+    isPostOnly (type, timeInForce, params = {}) {
         /**
          * @ignore
          * @method
          * @param {string} type Order type
          * @param {string} timeInForce
-         * @param {boolean} exchangeSpecificOption True if the exchange specific post only setting is set
          * @param {dict} params Exchange specific params
          * @returns {boolean} true if a post only order, false otherwise
          */
         let postOnly = this.safeValue2 (params, 'postOnly', 'post_only', false);
-        params = this.omit (params, [ 'post_only', 'postOnly' ]);
```

Comment:
removing this is prone to error, the method should perform all the tasks that it needs to and there shouldn't need to be other code written when this method is called